templates/products/products.js

```diff
@@ -190,18 +190,30 @@ const groupBy = (array, key) => array.reduce((accum, current) => {
   return accum;
 }, {});
 
+let locale = '';
```

Comment:
Since we are already passing these variables to fetchProducts, can we please move them inside decorate(doc) and avoid more global variables here?